API/fleece/Fleece.h

```diff
@@ -484,6 +484,10 @@ while (NULL != (value = FLArrayIterator_GetValue(&iter))) {
         Its initial ref-count is 1, so a call to FLMutableArray_Free will free it.  */
     FLMutableArray FLMutableArray_New(void) FLAPI;
 
+    /** Creates a new mutable Array from JSON. The input json must represent a JSON array or NULL will be returned.
+        Its initial ref-count is 1, so a call to FLMutableArray_Free will free it.  */
```

Comment:
FLMutableArray_Release instead of FLMutableArray_Free?

Reply:
fixed